Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -201,6 +201,9 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         if indexPath.row + 1 >= collectionView.numberOfItems(inSection: 0) {
             channelController.loadPreviousMessages()
         }
+
+        // Force cell layout to prevent animation glitches
+        cell.layoutIfNeeded()
```

Comment:
Don't we need `setNeedsLayout` instead of `layoutIfNeeded`?

Reply:
Right, fixed in 176d54b